### PR TITLE
fix(cwspr): Fix activation failure in Q and toolkit

### PR DESCRIPTION
**Pull request by leigaol**
## Problem

feature/standalone is breaking. both extension cannot activate.

Root cause:

Q: globals.loginManager referenced before intialize().

Toolkit: makeLogger -> globals.contextPrefix  referenced before intialize().

## Solution

Fix activation by moving  global reference

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/core/src/codewhisperer/commands/startSecurityScan.ts [hunk 1]
```diff
@@ -266,6 +265,11 @@ function populateCodeScanLogStream(scannedFiles: Set<string>) {
     // Clear log
     securityScanOutputChannel.clear()
     const numScannedFiles = scannedFiles.size
+    if (codeScanLogger === undefined) {
+        codeScanLogger = makeLogger({
+            outputChannels: [securityScanOutputChannel],
+        })
+    }
     if (numScannedFiles === 1) {
```
[the remark this answers is not in the document]
Comment (leigaol) [reply packages/core/src/codewhisperer/commands/startSecurityScan.ts:273]: Blocked by 
>https://github.com/aws/aws-toolkit-vscode/pull/4656/files

[the remark this answers is not in the document]
Comment (justinmk3) [reply packages/core/src/codewhisperer/commands/startSecurityScan.ts:273]: unblocked. please rebase

File: packages/core/src/codewhisperer/commands/startSecurityScan.ts [hunk 2]
```diff
@@ -266,6 +265,11 @@ function populateCodeScanLogStream(scannedFiles: Set<string>) {
     // Clear log
     securityScanOutputChannel.clear()
     const numScannedFiles = scannedFiles.size
+    if (codeScanLogger === undefined) {
+        codeScanLogger = makeLogger({
+            outputChannels: [securityScanOutputChannel],
+        })
+    }
```
Comment (justinmk3) [inline packages/core/src/codewhisperer/commands/startSecurityScan.ts:272]: not needed after rebase

File: packages/amazonq/src/extensionShared.ts [hunk 3]
```diff
@@ -51,6 +51,7 @@ export async function activateShared(context: vscode.ExtensionContext) {
     await activateLogger(context, contextPrefix, qOutputChannel, qLogChannel)
     globals.outputChannel = qOutputChannel
     globals.logOutputChannel = qLogChannel
+    globals.loginManager = new LoginManager(globals.awsContext, new CredentialsStore())
```
Comment (justinmk3) [inline packages/amazonq/src/extensionShared.ts:54]: only this change is needed

**Pull request merged**